src/loaders/ObjectLoader.js

```diff
@@ -280,6 +280,14 @@ THREE.ObjectLoader.prototype = {
 
 				}
 
+				if ( data.aoMap !== undefined ) {
+
+					material.aoMap = getTexture( data.aoMap );
+					if ( data.aoScale ) {
```

Comment:
For clarity and robustness I would change this `if ( data. aoScale !== undefined )`.